cloudbuild/create-key-pair.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
this file copied from https://github.com/GoogleCloudDataproc/custom-images/tree/2024.11/examples/secure-boot